support/cas-server-support-couchdb-authentication/src/main/java/org/apereo/cas/authentication/CouchDbAuthenticationHandler.java

```diff
@@ -0,0 +1,22 @@
+package org.apereo.cas.authentication;
+
+import org.apereo.cas.authentication.principal.PrincipalFactory;
+import org.apereo.cas.integration.pac4j.authentication.handler.support.UsernamePasswordWrapperAuthenticationHandler;
+import org.apereo.cas.services.ServicesManager;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * This is {@link CouchDbAuthenticationHandler}.
+ *
+ * @author Timur Duehr
+ * @since 6.0.0
+ */
+@Getter
```

Comment:
Annotations should be removed here, if they are no-ops?

Reply:
Done